src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -2659,28 +2661,27 @@ namespace WaterToAirHeatPumpSimple {
 
         // Update heat pump data structure
         state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).Power = state.dataWaterToAirHeatPumpSimple->Winput;
-        state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).QLoadTotal = state.dataWaterToAirHeatPumpSimple->QLoadTotal;
+        state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).QLoadTotal = state.dataWaterToAirHeatPumpSimple->QLoadTotalReport;
```

Comment:
I see, these are 2 different variables, one in struct SimpleWatertoAirHPConditions and the other in state struct WaterToAirHeatPumpSimpleData.